docs/components/plugins/api/APISectionUtils.tsx

```diff
@@ -380,27 +393,72 @@ export const getTagData = (tagName: string, comment?: CommentData) =>
 export const getAllTagData = (tagName: string, comment?: CommentData) =>
   comment?.tags?.filter(tag => tag.tag === tagName);
 
+const getCleanPlatformName = (platform: CommentTagData) => {
```

Comment:
```suggestion
const getPlatformName = (platform: CommentTagData) => {
```

I don't think `clean` adds any helpful information to this function name